heimdall-frontend/source/mainwindow.cpp

```diff
@@ -659,7 +659,7 @@ void MainWindow::SelectPartitionName(int index)
 
 		partitionFileGroup->setTitle(title);
 
-		if (!fileInfo.GetFilename().isEmpty())
+		if (pitEntry && !fileInfo.GetFilename().isEmpty())
```

Comment:
Hmm, pitEntry should not be null here. If it is then there's a problem elsewhere. Did you encounter this personally?